tasks/Allocating Resources to Programs.md

```diff
@@ -0,0 +1,63 @@
+# Summary
+Participants were organized into groups of four and performed a series of tasks in a series of rounds with old and new group members, in some rounds team member names were visible (unmasked) while other rounds they were not (masked). Teams were given $500,000 and had to allocate funds between a number of competing programs which were each designed to appeal to particular personal values. 
+
+# References
+Main Paper: https://hci.stanford.edu/publications/2019/fracture/fracture-cscw2019.pdf
+
+Task Related: Richard T. Watson, Gerardine DeSanctis, and Marshall Scott Poole. 1988. Using a GDSS to Facilitate Group Consensus:
+Some Intended and Unintended Consequences. MIS Quarterly 12, 3 (1988), 463–478. http://www.jstor.org/stable/249214
+
+# Stimuli
+## The visual components
+![image](https://user-images.githubusercontent.com/78745728/113324863-9af75a80-92e5-11eb-80a3-cb9d2ade4a87.png)
+
```

Comment:
Either in stimuli or alternative materials you could add Table 2. in https://www.jstor.org/stable/249214?seq=7#metadata_info_tab_contents that has the personality value and corresponding project.